ddtrace/internal/ci_visibility/encoder.py

```diff
@@ -195,8 +196,15 @@ def _build_payload(self, traces):
     @staticmethod
     def _convert_span(span, dd_origin):
         # type: (Span, str) -> Dict[str, Any]
-        return {
+        converted_span = {
             "test_session_id": int(span.get_tag(SESSION_ID) or "1"),
             "test_suite_id": int(span.get_tag(SUITE_ID) or "1"),
             "files": json.loads(span.get_tag(COVERAGE_TAG_NAME))["files"],
         }
+
+        from ddtrace.internal.ci_visibility.recorder import _get_test_skipping_level
```

Comment:
Why not import this at the top of the module? This is getting called every time we call `_build_data()`, so it's not really a conditional import.

Reply:
It's due to a circular import, but I can move this function to a different place to avoid it.